### ISSUE TITLE
[Bug]: Column Seperator is not accessible by the keyboard

### ISSUE THREAD

**Issue by Sachin01256** [microsoft/fluentui#24883]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
Issue persist on Chrome, Edge and firefox browsers
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://codepen.io/sachin01256/pen/JjvyoEd?editors=0010

## Actual Behavior
Unable to drag the column header from left to right or right to left in the table using keyboard in DetailsList control, where as when we try do the same with mouse then it's working fine.
Keys using to drag:
Ctrl+Alt+Left Key   = Right to left
Ctrl+Alt+RightKey =  Left to right

## Expected Behavior
Expecting the behavior as given in attached video. On keyboard navigation, the user should be able to perform the drag operation from left to right or right to left for the column header. 

https://user-images.githubusercontent.com/107034474/191475073-b8851e87-e876-406c-bfe0-d53ea10f0032.mp4

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by TristanWatanabe** [comment]
This type of behavior is not currently built into `DetailsList` and most likely won't be in the near future as we're focused more on convergence work for v9. We were aware of the limitations with column resize and reorder via the keyboard which is why we created an example on how to accomplish said behavior for it here: https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/keyboardcolumnreorderresize.

**Comment by YingXue** [comment]
Hi @TristanWatanabe 
Thanks for providing the workaround for this accessibility issue.
However, while digging into the example code provided, I found that the columns are automatically generated using the function: b**uildColumns(items, ...)** where the columns are **automatically derived** from the item's properties.
With this being said, we no longer have the freedom to custom the column header text. Am I missing something?

Thanks.
